packages/blitz-dom/src/document.rs

```diff
@@ -222,6 +227,12 @@ impl DocumentLike for Document {
                             }
                             self.set_focus_to(node_id);
                         }
+                    } else if el.name.local == local_name!("a") {
+                        if let Some(href) = el.attr(local_name!("href")) {
```

Comment:
So this is working. But it only works if you click the actual `<a>` element. It doesn't work if you click a child within the `<a>` (even a text node). We should somehow implement event bubbling to make this work.

Reply:
Actually we have a bigger, somehow related problem here.

Depending on the page, when I click an icon next to a link or so, the assertion that the node we clicked on and the node that fired the event are the same doesn't hold anymore and blitz panics.

Locally I've just removed that assertion, but maybe we should check if they are descendants of one another or so? Not sure.

Also on phone right now, so I don't know where exactly it originates.

Reply:
> So this is working. But it only works if you click the actual `<a>` element. It doesn't work if you click a child within the `<a>` (even a text node). We should somehow implement event bubbling to make this work.

```
thread 'main' panicked at packages/blitz-dom/src/document.rs:183:21:
assertion failed: hit.node_id == event.target
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I guess that's what you meant. I shouldn't write comments while doing groceries :D

Reply:
I'm definitely hitting a case where nothing happens rather than a panic (open `https://servo.org` and try clicking the text in the header links). How are you triggering the panic?